src/Command.php

```diff
@@ -511,6 +521,18 @@ public function execute()
                         }
                         fclose($pipes[0]);
                     }
+
+                    if($this->streamOutput)
+                    {
+                        while(!feof($pipes[1]))
+                        {
+                            $return_message = fgets($pipes[1], 1024);
+                            if (strlen($return_message) == 0) break;
+
+                            echo $return_message;
+                        }
+                    }
```

Comment:
This block would not work as expected: In blocking mode, this part is not reached until the command ist complete. If we want to introduce such a feature it only makes sense for `nonBlockingModeˇ.